client/my-sites/stats/stats-insights/index.jsx

```diff
@@ -69,8 +70,10 @@ export default React.createClass( {
 							beforeNavigate={ this.updateScrollPosition } />;
 		}
 
+		// TODO: should be refactored into separate components
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
Yeah, I see the warnings as a nag, but a legitimate one. We should try to fix it, and if we can't in the same pull request, we should be comfortable in leaving it as a warning to be addressed later. Disabling should be reserved for cases where we have no intention of ever changing it to adhere to the rule.